components/TechGala/ProjectCard.js

```diff
@@ -1,11 +1,12 @@
+import Image from 'next/image';
 import React from 'react';
 
 function ProjectCard(props) {
     const {image, title, winner, summary, names, url, category} = props.project;
     return (
         <div className="intro-row">
             <div className="show-mobile">
-                <img src={image} alt={`showcase image for ${title}`}/>
+                <Image src={image} alt={`showcase image for ${title}`} height={280} width={370} />
```

Comment:
Maybe a dumb question, but it seems like height and width are now required props for images; what are the implications for responsiveness?

Reply:
The images automatically readjust themselves based on the size of the page, iirc. The height and width props are for the general case.

Reply:
Yeah, I think moving forward @advaithg is going to be the SME for `next/image` - you may want to discuss how the component works (ex specified with/height versus `layout="fill"`, what's happening under the hood) with the team some time.